sdk/cosmos/azure-cosmos/README.md

```diff
@@ -602,6 +602,31 @@ even when it isn't enabled for the client:
 database = client.create_database(DATABASE_NAME, logging_enable=True)
 ```
 
+Alternatively, you can log using the CosmosHttpLoggingPolicy, which extends from the azure core HttpLoggingPolicy, by passing in your logger to the `logger` argument.
```

Comment:
The HttpLoggingPolicy is a developer concept - not really an end user concept - so I would not refer to it by name, just discuss the keyword argument.